src/plays/wordle/components/WordleRow.tsx

```diff
@@ -26,6 +26,7 @@ export default function WordleRow(props: { tileRow: TileRow; wordleWord: string
                   roll
                   correct={isCorrect}
                   index={index}
+                  key={`${index}-${letter}`}
```

Comment:
Never use index in key, can we just use the letter?

Reply:
the letter might be an empty string. can we use `uuid` instead?

Reply:
Yes for now use uuid.

Reply:
Done!!